browser/src/Editor/NeovimEditor/WelcomeBufferLayer.tsx

```diff
@@ -553,74 +590,74 @@ export interface IWelcomeCommandsViewProps extends Partial<WelcomeViewProps> {
-        const { commands, executeCommand } = this.props
-        const isSelected = (command: string) => command === this.props.selectedId
+        const { commands } = this.props
         return (
             <LeftColumn>
                 <AnimatedContainer duration="0.25s">
                     <SectionHeader>Quick Commands</SectionHeader>
                     <WelcomeButton
                         title="New File"
-                        onClick={() => executeCommand(commands.openFile.command)}
+                        onClick={() => commands.openFile.execute()}
                         description="Control + N"
                         command={commands.openFile.command}
-                        selected={isSelected(commands.openFile.command)}
+                        selected={this.isSelected(commands.openFile.command)}
                     />
                     <WelcomeButton
                         title="Open File / Folder"
-                        onClick={() => executeCommand(commands.openWorkspaceFolder.command)}
                         description="Control + O"
+                        onClick={() => commands.openWorkspaceFolder.execute()}
                         command={commands.openWorkspaceFolder.command}
-                        selected={isSelected(commands.openWorkspaceFolder.command)}
+                        selected={this.isSelected(commands.openWorkspaceFolder.command)}
                     />
                     <WelcomeButton
                         title="Command Palette"
-                        onClick={() => executeCommand(commands.commandPalette.command)}
+                        onClick={() => commands.quickOpenShow.execute()}
                         description="Control + Shift + P"
-                        command={commands.commandPalette.command}
-                        selected={isSelected(commands.commandPalette.command)}
+                        command={commands.quickOpenShow.command}
+                        selected={this.isSelected(commands.quickOpenShow.command)}
                     />
                     <WelcomeButton
                         title="Vim Ex Commands"
                         description=":"
                         command="editor.openExCommands"
-                        onClick={() => executeCommand(commands.commandline.command)}
-                        selected={isSelected(commands.commandline.command)}
+                        onClick={() => commands.commandline.execute()}
+                        selected={this.isSelected(commands.commandline.command)}
                     />
                 </AnimatedContainer>
                 <AnimatedContainer duration="0.25s">
                     <SectionHeader>Learn</SectionHeader>
                     <WelcomeButton
                         title="Tutor"
-                        onClick={() => executeCommand(commands.openTutor.command)}
+                        onClick={() => commands.openTutor.execute()}
```

Comment:
This also doesn't seem to be working for me?

Not sure why, I think it used to, but its possible the command has been renamed.

Reply:
@CrossR this I believe never worked I don't think the functionality exists when I started refactoring it I noted it didn't work but also didn't know what it was supposed to do. Given the stated goals of oni having something related to that seemed important to keep on the front page, but it wasn't clear what that should be and tbh if the command doesn't already exist I think implementing it from scratch is beyond the scope of what I wanted to do here

Reply:
Its the one thats left but I couldn't find a quick solution and realllly didnt want to remove it, could always put a coming soon on it or disable the button but not sure we have a template for styling a disabled button

Reply:
Thats fair enough, we can look at fixing up the `experimental.tutorial.start` command I've found in a different PR. Might make a good "first issue" PR, just need to look into the scope of it.

Or, it can be swapped to be like the "Toggle Explorer" command, to pop out the tutorial side bar.

Reply:
Or open up your current/last tutorial? think those are all good things it could do, a good first issue sounds like a good idea since we can then just point the person to the example with the explorer and the other panes that have that functionality